lib/prometheus.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Prometheus = require('prom-client');
+
+function makePrometheusClient() {
+    return Prometheus;
+}
+
+module.exports = makePrometheusClient;
```

Comment:
What is the point of this file?

Reply:
Something left over from pattern-matching.  Removed in latest.